hydra-node/test/Hydra/BehaviorSpec.hs

```diff
@@ -393,12 +428,13 @@ waitUntil nodes expected =
 -- | A thin layer around 'HydraNode' to be able to 'waitFor'.
 data TestHydraNode tx m = TestHydraNode
   { send :: ClientInput tx -> m ()
+  , chainEvent :: ChainEvent tx -> m ()
```

Comment:
Seems like the implementation ignores `Observation` chain events (makes sense), so I would suggest we make it obvious that we only can issue rollbacks with this function by using the signature:

```hs
, chainRollback :: Word -> m ()
```

or whatever the payload of `Rollback` be by the time you reading this.